blazor/datagrid/column-menu.md

```diff
@@ -70,6 +70,83 @@ The default menu items are displayed in the following table,
 The following image represents DataGrid with column menu property enabled,
 ![Blazor DataGrid with Column Menu](images/blazor-datagrid-column-menu.gif)
 
+## Custom column menu
+
+You can customize the column menu by using the [ColumnMenuItems](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_ColumnMenuItems) property and the [ColumnMenuItemClicked](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_ColumnMenuItemClicked) event of the Grid.
+
+In the following sample, we have changed the freeze direction of the column using the custom column menu.
```

Comment:
In the following sample, we have changed the freeze direction of the column using the custom column menu: